libs/pika/execution_base/tests/unit/any_sender.cpp

```diff
@@ -625,8 +625,10 @@ struct wait_globals
 
 void test_globals()
 {
-    // TODO: No ensure_started implementation in reference implementation.
-#if !defined(PIKA_HAVE_STDEXEC)
+    // No implementation of unique_any_sender in stdexec
+#if defined(PIKA_HAVE_STDEXEC)
+    global_any_sender = std::move(global_any_sender) | ex::ensure_started() | ex::split();
+#else
```

Comment:
There's also no `any_sender` in stdexec. There is `exec::any_sender_of` but we don't pull that in to `pika::execution::experimental`.

Basically, both of these tests should be possible to enable unconditionally now (modulo bugs).